packages/@aws-cdk/aws-events-targets/test/sqs/sqs.test.ts

```diff
@@ -140,3 +142,43 @@ test('multiple uses of a queue as a target results in multi policy statement bec
     Queues: [{ Ref: "MyQueueE6CA6235" }]
   }));
 });
+
+test('fail if MessageGroupId is specified on non-fifo queues', () => {
+  const stack = new Stack();
+  const queue = new sqs.Queue(stack, 'MyQueue');
+
+  throws(() => new targets.SqsQueue(queue, { messageGroupId: 'MyMessageGroupId' }),
```

Comment:
Since this is jest I think you should use jest's `expect(() => x).toThrow(...)`. This will require renaming the `@aws-cdk/assert`'s `expect` to something else.

Reply:
Don't even  understand how it's possible to import `expect` from `@aws-cdk/assert` in a jest test at all.

The correct import would be:

```ts
import '@aws-cdk/assert/jest';
```